src/theory/quantifiers/theory_quantifiers_type_rules.cpp

```diff
@@ -143,6 +143,25 @@ TypeNode QuantifierInstPatternListTypeRule::computeType(
   }
   return nodeManager->instPatternListType();
 }
+TypeNode QuantifierOracleFormulaGenTypeRule::computeType(
+    NodeManager* nodeManager, TNode n, bool check)
+{
+  Assert(n.getKind() == kind::ORACLE_FORMULA_GEN);
+  if (check)
```

Comment:
we don't need to check the arity of `n`?

Reply:
Like other kinds, this is done automatically via the kinds infrastructure.